src/WinRT.Runtime/ObjectReference.cs

```diff
@@ -167,34 +167,78 @@ public unsafe TInterface AsInterface<TInterface>()
 
         public int TryAs<
 #if NET
-        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.NonPublicConstructors | DynamicallyAccessedMemberTypes.PublicFields)]
+            [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.NonPublicConstructors | DynamicallyAccessedMemberTypes.PublicFields)]
 #endif
-        T>(out ObjectReference<T> objRef) => TryAs(GuidGenerator.GetIID(typeof(T)), out objRef);
+            T>(out ObjectReference<T> objRef) => TryAs(GuidGenerator.GetIID(typeof(T)), out objRef);
 
-        public virtual unsafe int TryAs<
+        public unsafe int TryAs<

```

Comment:
If someone doesn't override the `TryAs` function but does extend this class, does this actually break anything or no?